packages/wonder-blocks-tooltip/components/tooltip-anchor.test.js

```diff
@@ -429,33 +421,41 @@ describe("TooltipAnchor", () => {
         test("active state was stolen, active is set to false immediately", async () => {
             // Arrange
             let wrapper;
+            let activeState = false;
             const ref = await new Promise((resolve) => {
-                const getFakeTooltipPortalMounter = (active) =>
-                    (((
-                        <View>{active ? "true" : "false"}</View>
-                    ): any): React.Element<typeof TooltipPortalMounter>);
                 const nodes = (
-                    <TooltipAnchor anchorRef={resolve}>
-                        {getFakeTooltipPortalMounter}
-                    </TooltipAnchor>
+                    <View>
```

Comment:
The `View` isn't necessary... I was copy pasting from an example where it was used.  I'm going to get rid of it so we don't have unnecessary code changes in this PR.